demos-europe/demosplan-core
# F t33175 foundry

**Ticket:** https://yaits.demos-deutschland.de/t33175

To improve the test-writing-experience, we want to use foundry.
To do so, this PR includes the setup of foundry and includes an example usage as well as the related documentation

### How to review/test
Execute the TagServcieTest.php and maybe some others.
Check the documentation for comprehensibility.
